### DISCUSSION TITLE
Supercell

### DISCUSSION THREAD

**Discussion by Meso22** [niels747/2D-Weather-Sandbox#12] [Q&A] [answered]
What conditions are needed to create a supercell.

**Comment by Meso22** [comment, marked as answer]
Never Mind I figured it out.

**Reply by Westerstaad** [reply]
What are the conditions required?

**Reply by Meso22** [reply]
What I did was set the water temperature to max 35 and then left it for a few in game days until the air temperature was about 30c, after that I set the wind to 0.3 and the evaporation heat to 2.5. If you want to have more hail set the freezing and melting rate in the precipitation settings to 0.0005.

**Reply by Westerstaad** [reply]
Ill try that

**Reply by Meso22** [reply]
I forgot to mention the supercells last longer when the map size is 6000 it is just much slower

**Reply by Westerstaad** [reply]
you can just disable auto adjust but the frames wont be smooth
